ros/src/models/chrono/traj_gen_chrono/src/path_gen_chrono.cpp

```diff
@@ -38,28 +38,20 @@ int main(int argc, char **argv) {
 
     int count = 0;
     while (ros::ok()) {
-        ROS_INFO("Hello world");
-
-        // if (count < path->getNumPoints()-1) {
-        //     ChVector<> pos_1 = path->getPoint(count);
-        //     ChVector<> pos_2 = path->getPoint(count+1);
-
-        //     control_info.x = {pos_1[0], pos_2[0]};
-        //     control_info.y = {pos_1[1], pos_2[1]};
-
-        //     count++;
-        // }
-
         count++;
-        if (count % 2)
-            control_info.y =  {-125.0000, -125.0000};
-        else
+        if (count % 2){
+            ROS_INFO("Swithed to right lane");
             control_info.y =  {-120.0000, -120.0000};
```

Comment:
The original objective of the path_gen_chrono.cpp is to test the functionality of path_follower.cpp by sending dynamic paths to it. This cpp is not written in a very general way because in the future the path_follower.cpp will receive the path directly from the nloptcontrol_planner instead of this path_gen_chrono.cpp. But making all parameters of this cpp into YAML files indeed provides an alternative way of sending paths. I'll improve it.

Reply:
thank you. In general, we want to develop functions not hard coded examples